src/main/java/org/datadog/jmxfetch/JmxAttribute.java

```diff
@@ -191,6 +194,22 @@ private List<String> getBeanParametersList(
         return beanTags;
     }
 
+    private List<String> normalizeInstanceParameters(List<String> beanParametersList) {
```

Comment:
Can we consolidate this functionality into the existing `sanitizeParameters` function? They're both doing similar things at a high level -- iterating through a list of bean parameters and formatting them into tags.

Reply:
we discussed this, `sanitizeParameters` is `static` which is why this was separated. Lets try to make `sanitizeParameters` non-static and consolidate.